src/cmd/oui/create.rs

```diff
@@ -0,0 +1,112 @@
+use super::{map_addresses, print_txn};
+use crate::{
+    cmd::*,
+    keypair::PublicKey,
+    result::Result,
+    traits::{TxnEnvelope, TxnFee, TxnSign, TxnStakingFee},
+};
+use helium_api::{ouis, Client};
+use structopt::StructOpt;
+
+/// Allocates an Organizational Unique Identifier (OUI) which
+/// identifies endpoints for packets to sent to The transaction is not
+/// submitted to the system unless the '--commit' option is given.
+#[derive(Debug, StructOpt)]
+pub struct Create {
+    /// The address(es) of the router to send packets to
+    #[structopt(long = "address", short = "a", number_of_values(1))]
+    addresses: Vec<PublicKey>,
+
+    /// Optionally request a specific OUI. They must increment
+    /// from previous OUI by 1.
```

Comment:
Is this document string still correct?

Reply:
Good catch - it seems wrong. Changed it

Reply:
Actually, I think what I was trying to warn people is still correct. I tried to rephrase the doc to make it more clear.